challenges/objects-arrays.js

```diff
@@ -51,23 +72,22 @@ const graduates = [
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities);
-
+const universities = graduates.map((graduate) => graduate.university);
+console.log(universities.sort());
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. This will be an array of strings.
 
 The resulting contact information strings should have a space between the first name and the email, like this: 
 "Josh josh@example.com"
 
 Log the result of your new array. */
 const contactInfo = [];
+graduates.forEach((graduate) => contactInfo.push(graduate.first_name + ' ' + graduate.email));
 console.log(contactInfo);
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called unisWithUni that contains them all. This will be an array of objects. Log the result. */
-const unisWithUni = [];
+const unisWithUni = graduates.filter(graduate => graduate.university.includes("Uni"));
 console.log(unisWithUni);
```

Comment:
You were required to return just an array containing the universities that match the specification, rather than the whole objects.